apps/comps/components/competitions-table.tsx

```diff
@@ -222,7 +222,7 @@ export const CompetitionsTable: React.FC<CompetitionsTableProps> = ({
                       colSpan={header.colSpan}
                       style={{ width: header.getSize() }}
                       onClick={header.column.getToggleSortingHandler()}
-                      isSorted={header.column.getIsSorted()}
+                      sortState={header.column.getIsSorted() ? "asc" : "none"}
```

Comment:
```suggestion
                      sortState={
                        header.column.getIsSorted() === "asc"
                          ? "asc"
                          : header.column.getIsSorted() === "desc"
                            ? "desc"
                            : "none"
                      }
```

Reply:
in the agents-table this is needed as well. Can you create a helper to map this values on both places?

https://github.com/recallnet/js-recall/pull/531/files#diff-66affd67427ee349ab1993083adf791b0ede33009977c3fd66da8cb79593e708L255-R256